go.mod

```diff
@@ -1,21 +1,28 @@
 module github.com/segmentio/chamber/v2
 
 require (
-	github.com/aws/aws-sdk-go v1.38.22
-	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/magiconair/properties v1.8.0
+	github.com/aws/aws-sdk-go v1.44.255
+	github.com/magiconair/properties v1.8.7
 	github.com/pkg/errors v0.9.1
-	github.com/segmentio/backo-go v0.0.0-20160424052352-204274ad699c // indirect
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.2 // indirect
+	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.2.2
-	github.com/xtgo/uuid v0.0.0-20140804021211-a0b114877d4c // indirect
-	golang.org/x/sys v0.0.0-20220614162138-6c1b26c55098
-	gopkg.in/segmentio/analytics-go.v3 v3.0.1
+	golang.org/x/sys v0.7.0
+	gopkg.in/segmentio/analytics-go.v3 v3.2.1
 	gopkg.in/yaml.v3 v3.0.1
 )
 
-go 1.13
```

Comment:
Oh wow I definitely recall @rikez doing work to update golang version in a bunch of places but it's possible the go.mod file was left still on the old version. Still, I'm no longer on the team and will have them confirm if they're comfortable jumping to 1.20 or prefer to stay a version behind or not.

Reply:
I highly suspect it was just a mistake to leave the go.mod file with the older version. But I could be wrong.

Reply:
@asaf-erlich @nitrocode there are ongoing discussions about the state of `go 1.13` in the `go.mod` file

https://github.com/segmentio/chamber/pull/371

@knksmith57 has some context and we've been discussing internally, but unfortunately haven't prioritized. 

TL;DR - by updating the go version in `go.mod`, its a potentially breaking change for any GitHub repos that are consuming `chamber` as a dependency (which is an option).